### PR TITLE
Adding Checksums to artifactory call

### PR DESCRIPTION
**Pull request by brgaulin**
Note: This is not ready for merging. I have opened a few PRs against the javascript library we are using here to talk to artifactory. Once those get merged and released(or if they get stuck, I can fork), I will ping back here.

cgadam/artifactory-api#7

### CODE REVIEW

**Comment by brgaulin** [comment]
Note: This is not ready for merging. I have opened a few PRs against the javascript library we are using here to talk to artifactory. Once those get merged and released(or if they get stuck, I can fork), I will ping back here.

https://github.com/cgadam/artifactory-api/pull/7

File: index.js [hunk 1]
```diff
@@ -19,14 +21,16 @@ var publish_file = function (artifactory, repo_key, project, file, force_upload)
     if (file.indexOf('pom') > -1) { basename = project.artifact_id + '-' + project.version + '.pom'; }
 
     winston.info('Uploading ' + file + ' as ' + basename + ' into ' + repo_key);
-    return artifactory.uploadFile(repo_key, '/' + replace_dots(project.group_id) + '/' + project.artifact_id + '/' + project.version + '/' + basename, file, force_upload)
-    .then((uploadInfo) => {
+    get_checksums(file).then((checksums) => {
```
Comment (athieriot) [inline index.js]: Don't forget to bubble the promise upstream by returning it

Comment (athieriot) [inline index.js]: Don't forget to return the promise to bubble it upstream

**Comment by tboerger** [comment]
LGTM

**Comment by ganeshksankar** [comment]
This is the exact way that I'm looking for artifactory uploads with checksum. Could you please merge this PR if it looks good

**Comment by athieriot** [comment]
I'll try to ping the guys from cgadam/artifactory-api#7 if they can release a new version

**Pull request closed without merging**